content/blog/kubecon-na-2022/index.md

```diff
@@ -0,0 +1,43 @@
+---
+path: '/kubecon-na-2022'
+title: 'Come meet us at Kubecon NA 2022'
+date: 2022-10-18T00:00:00.000+00:00
+featured_image: hero-image.png
+categories: ['Pixie Team Blogs', 'Announcements', 'Kubernetes']
+authors: ['Zain Asgar', 'Michelle Nguyen']
+emails: ['zasgar@pixielabs.ai', 'michelle@pixielabs.ai']
+---
+
+Pixie contributors will be attending this year’s [KubeCon + CloudNativeCon North America 2022](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/) in Detroit, Michigan!
+
+Come chat about open source observability for Kubernetes with Pixie’s core contributors at our booth in the [Project Pavilion](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/project-engagement/#project-pavilion).
+
+Pixie contributors will also be speaking at the following events:
+
+- [**OpenTelemetry or eBPF? That is the Question**](https://sched.co/1Auyh) ([eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/) - Monday, October 24, 2022 9:25 AM EDT)<br/><br/>
+_In the observability space, OpenTelemetry and eBPF are two technologies that have been rapidly changing the landscape. So which should you use? In this session, we'll cover the strengths and weaknesses of both approaches, and show how both approaches have a role to play._
+- [**Tracing SSL/TLS Encrypted Microservices with eBPF**](https://sched.co/1Auyt) ([eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/) - Monday, October 24, 2022 11:15 AM EDT)<br/><br/>
```

Comment:
Sorry for nitpicking. :P Maybe try with just one `<br />`? The thing that looks weird now is that this space between the title and description is now larger than the space between the bullet points.